pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -30,7 +30,7 @@ const builtinRegistriesConfPath = "/etc/containers/registries.conf"
 // systemRegistriesConfDirPath is the path to the system-wide registry
 // configuration directory and is used to add/subtract potential registries for
 // obtaining images.  You can override this at build time with
-// -ldflags '-X github.com/containers/image/sysregistries.systemRegistriesConfDirecotyPath=$your_path'
+// -ldflags '-X github.com/containers/image/sysregistries.systemRegistriesConfDirectoryPath=$your_path'
```

Comment:
Note to self:
```suggestion
// -ldflags '-X github.com/containers/image/sysregistries.systemRegistriesConfDirPath=$your_path'
```
, or rather `containers/image/v5/…` IIRC, actually. That’s out of scope though.

Reply:
I think I'm going to leave this to you, it's too complicated to count as spelling.